packages/vendure-plugin-invoices/src/ui/invoices-nav.module.ts

```diff
@@ -61,6 +65,16 @@ import { ApolloCache } from '@apollo/client/cache';
             (event.target as HTMLButtonElement).disabled = false;
           });
       },
+      buttonState: (context) => {
+        const orderId = context.route.snapshot.params['id'];
```

Comment:
Minor possible improvement: i remember something about the entity (the order in this case) is available on the context object. If thats the case we dont need to do an extra query

Reply:
I think `ActionBarContext` only has the following fields: 
```ts
{
    route: ActivatedRoute;
    injector: Injector;
    dataService: DataService;
    notificationService: NotificationService;
}
```